src/resolver/dependency.rs

```
@@ -310,3 +321,31 @@ impl fmt::Display for UnresolvedDependency<'_> {
         )
     }
 }
+
+fn query_r_universe(package: &Package, repo_url: &str) -> Result<(String, String), ()> {
```

We want to return an actual error so we can display/log it